### ISSUE TITLE
Fails to compile time crate

### ISSUE THREAD

**Issue by jhutchins** [xd009642/tarpaulin#505]
I wish that I could say more about the specifics of why it is failing, but even with the verbose flag there's not much in the way of interesting output. This is what is shown when compiling using the latest official tarpaulin docker image (0.14.2) and the most minimal of examples.

```
$ docker run --security-opt seccomp=unconfined -v "$PWD:/volume" xd009642/tarpaulin:latest cargo tarpaulin --verbose
[INFO tarpaulin] Creating config
[INFO tarpaulin] Running Tarpaulin
[INFO tarpaulin] Building project
   Compiling time v0.2.16
error: could not compile `time`.

To learn more, run the command again with --verbose.
[ERROR tarpaulin] Failed to compile tests! Error: cannot find attribute `skip` in this scope
Error: "Failed to compile tests! Error: cannot find attribute `skip` in this scope"
```

For this example I simply init an empty project and added time as the only dependency. The directory layout looks like this
```
.
├── Cargo.lock
├── Cargo.toml
└── src
    └── main.rs
```
where the contents of Cargo.toml is
```
[package]
name = "tmp"
version = "0.1.0"
authors = ["Jeff Hutchins"]
edition = "2018"

# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html

[dependencies]
time = "0.2.16"
```
and the contents of src/main.rs is
```
fn main() {
    println!("Hello, world!");
}
```
It should be noted that this compiles just fine using the official rust image.

**Comment by xd009642** [comment]
So the solution is either to move down to tarpaulin 0.13.x, got to a github link for the time version or wait for the next time release. They've fixed it in tree but from what I understand it won't be released until 0.3 is ready time-rs/time#267. It's down to a breaking change in 0.14 tarpaulin which was needed to make the attributes more sophisticated.

**Comment by jhpratt** [comment]
FYI to avoid more changes than necessary when switching from 0.2 to 0.3, I've recently backported a large number of commits from the 0.3 branch. You should be able to use tarpaulin with 0.2 as of last week. As I also removed the `#[inline]` attributes, I can now get reasonably accurate coverage (just a few false negatives).
